src/dpp/webhook.cpp

```diff
@@ -33,6 +33,25 @@ webhook::webhook() : managed(), type(w_incoming), guild_id(0), channel_id(0), us
 {
 }
 
+webhook::webhook(std::string webhook_url)
+{
+	token = webhook_url.substr(webhook_url.find_last_of("/") + 1);
+	try
+	{
+		id = std::stoull(webhook_url.substr(sizeof("https://discord.com/api/webhooks/") - 1, webhook_url.find_last_of("/")));
```

Comment:
this isn't doing what you think it is, sizeof() on a std::string will return the size of the object not the string length.
use webhook_url.length(), or for a const char*, strlen.